### PR TITLE
Add Radius CLI login support on brocade ICX series

### PR DESCRIPTION
**Pull request by lzammit**
# Description

Add Radius CLI login support on brocade ICX series. It uses the Radius attribute Foundry-Privilege-Level.

You need to assign the Write or Read access via an Administration role
# Impacts

NONE
# Delete branch after merge

YES 
# NEWS file entries
## New Features
- Brocade CLI authentication over RADIUS

### CODE REVIEW

**Comment by lzammit** [comment]
Tested on Brocade ICX 6450

**Comment by fdurand** [comment]
It's just miss in the documentation how to enable cli access over radius on the switch side.

File: lib/pf/Switch/Brocade.pm [hunk 1]
```diff
@@ -354,6 +354,47 @@ sub ifIndexToLldpLocalPort {
...
+=item returnAuthorizeWrite
+
+Return radius attributes to allow write access
+
+=cut
+
+sub returnAuthorizeWrite {
+   my ($self, $args) = @_;
+   my $logger = $self->logger;
+   my $radius_reply_ref;
```
Comment (jrouzierinverse) [inline lib/pf/Switch/Brocade.pm]: Initialize before use
`my $radius_reply_ref = {};`

File: lib/pf/Switch/Brocade.pm [hunk 2]
```diff
@@ -354,6 +354,47 @@ sub ifIndexToLldpLocalPort {
...
+=item returnAuthorizeRead
+
+Return radius attributes to allow read access
+
+=cut
+
+sub returnAuthorizeRead {
+   my ($self, $args) = @_;
+   my $logger = $self->logger;
+   my $radius_reply_ref;
```
Comment (jrouzierinverse) [inline lib/pf/Switch/Brocade.pm]: Initialize before use
`my $radius_reply_ref = {};`

**Comment by lzammit** [comment]
Doc at #1634

**Pull request merged**